app/javascript/v3/views/auth/signup/Index.vue

```diff
@@ -1,24 +1,56 @@
 <template>
-  <div class="h-full w-full dark:bg-slate-900">
-    <div v-show="!isLoading" class="flex h-full">
+  <div class="flex h-full w-full dark:bg-slate-900 relative overflow-hidden">
+    <div
+      class="absolute inset-0 h-full w-full bg-white dark:bg-slate-900 bg-[radial-gradient(var(--w-100)_1px,transparent_1px)] dark:bg-[radial-gradient(var(--w-800)_1px,transparent_1px)] [background-size:16px_16px] z-0"
+    />
+    <div
+      class="absolute h-full w-full overlay-gradient top-0 left-0 blur-[3px]"
+    />
+
+    <svg
+      width="528"
+      height="96"
+      viewBox="0 0 528 96"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      class="absolute -left-full -bottom-full"
+    >
+      <defs>
+        <mask id="cutoutMask">
+          <path
+            d="M263.999 65.0769C293.048 65.0769 317.641 45.9499 325.929 19.5735C329.219 9.10348 337.794 0 348.744 0H508.172C519.122 0 527.999 8.89875 527.999 19.8759V95.5C527.999 95.5 519.122 95.5 508.172 95.5H263.999H-0.000976562V19.8759C-0.000976562 8.89877 8.87604 0 19.8264 0H179.254C190.204 0 198.779 9.10349 202.069 19.5735C210.357 45.9499 234.95 65.0769 263.999 65.0769Z"
+            fill="white"
+          />
+        </mask>
+      </defs>
+    </svg>
+
+    <div
+      class="flex w-full h-full relative z-50 items-center justify-center overflow-auto"
+    >
       <div
-        class="flex-1 min-h-[640px] inline-flex items-center h-full justify-center overflow-auto py-6"
+        class="relative flex-1 min-h-[640px] inline-flex items-center h-full justify-center py-6"
       >
-        <div class="px-8 max-w-[560px] w-full overflow-auto">
-          <div class="mb-4">
+        <div
+          class="relative bg-white dark:bg-slate-800 before:bg-white before:border-slate-200 dark:before:border-slate-700 dark:before:bg-slate-800 px-16 pt-8 pb-16 max-w-[528px] w-full rounded-3xl signup-box"
+        >
+          <div class="absolute -top-[116px] right-0 left-0 w-24 h-24 mx-auto">
             <img
-              :src="globalConfig.logo"
+              :src="globalConfig.logoThumbnail"
```

Comment:
Please use logo as a fallback.

Reply:
I didn't get this comment. Since this is square container, if they don't have it setup, we could totally hide the logo. It'll look like this. 

 <img width="919" alt="image" src="https://github.com/chatwoot/chatwoot/assets/1277421/eeda8545-1028-4049-adcc-db11c62ced0a">